vnext/ReactUWP/Modules/NetworkingModule.cpp

```diff
@@ -336,11 +339,17 @@ void NetworkingModule::NetworkingHelper::SendRequest(
 }
 
 void NetworkingModule::NetworkingHelper::AbortRequest(int64_t requestId) noexcept {
-  auto iter = m_requests.find(requestId);
-  if (iter == end(m_requests))
-    return;
+  winrt::Windows::Foundation::IAsyncOperationWithProgress<
+    winrt::Windows::Web::Http::HttpResponseMessage, 
+    winrt::Windows::Web::Http::HttpProgress> httpRequest(nullptr);
+  {
+    std::scoped_lock(m_mutex);
```

Comment:
>( [](start = 20, length = 1)

There must be a variable name for the lock.

Reply:
Ahh good catch. Fixed.

---
In reply to: [384211714](https://github.com/microsoft/react-native-windows/pull/4179#discussion_r384211714) [](ancestors = 384211714)